lib/validator/v3/validator_test.go

```diff
@@ -481,6 +481,10 @@ func init() {
 		Entry("should reject a named port KubeNodePortRanges value", api.FelixConfigurationSpec{KubeNodePortRanges: &[]numorstring.Port{
 			numorstring.NamedPort("testport"),
 		}}, false),
+		Entry("should accept a valid list of ExternalNodesCidrList", api.FelixConfigurationSpec{ExternalNodesCidrList: &[]string{"1.1.1.1", "1.1.1.2/32", "1.1.3.0/23", "abcd::1", "abef::2/128"}},
```

Comment:
We shouldn't accept IPv6 CIDRs here because Felix will fail to put those in the all-hosts IP set.  IPIP is an IPv4 protocol anyway, at least the variant that we support is.

Reply:
sgtm. i'll make this change.